src/core/misc.c

```diff
@@ -261,6 +261,26 @@ char *gslist_to_string(GSList *list, const char *delimiter)
 	return ret;
 }
 
+gboolean gslist_is_unique(GSList *list)
+{
+	const char *prev;
```

Comment:
Why is this `const`? You should be able to remove the constness and avoid doing the type-casts below.